db_release_scripts/README.MD

```diff
@@ -1,144 +1,145 @@
+### DB Deployment Revoke 
 
-<em> **Enter:-bash-4.2$** cd /home/madmin/database_release/<<database_name>> </em>
+**Execute only if db version release deployment revoking required.**
 
-<em> **Enter:-bash-4.2$** bash bash <<schema>>_revoke_db_deploy.sh  <<schema>>_release_deploy.properties <release_version_number> </em>
+**Step 1** -> Run the DB Revoke deployment **"mosip_revoke_commons_db_deployment.sh "** script with release version as parameter.To access and exeute **"mosip_revoke_commons_db_deployment.sh "** script, follow the below given commands:
 
-**Step 2** -> Please observe <em> Post Release Deployment Validation </em>  steps below
+```
+bash-4.2$** cd /home/madmin/database_release/<<database_name>> 
 
+bash-4.2$** bash bash <<schema>>_revoke_db_deploy.sh  <<schema>>_release_deploy.properties <release_version_number>
+```
 
 ### Post Deployment Validation
 
-**Note:** If you encounter the following messages then please recheck the details(ip address, port number, database name, password) entered in the properties file, the message would be as follows:
+ Recheck the details(ip address, port number, database name, password) in the properties file in case of any following error messages:
 
+```
 <psql: could not translate host name "52.172.12.285" to address: Name or service not known>.
 
 <psql: FATAL:  password authentication failed for user "postgress">
 
 <psql: FATAL:  database "postgress" does not exist>
+```
+### Key points during or after the script execution
 
-**Key points during or after the script execution:**
+ * Properties file found message.
 
- * Properties file found message
+ * Server status.
 
- * Server status
+ * Accessing the right path to deploy DB.
 
- * Accessing the right path for DB deploy
+ * Creates respective roles.
 
- * Creates respective roles
+ * Check for any active connections.
 
- * Check for any active connections
-
- * Creates roles, creating Database, schemas, granting access, creating respective tables.
+ * Creates roles, creating database, schemas, granting access, creating respective tables.
 
  * Loading data or DML operations valid only for those DB's which carries DML actions.
 
  * End of sourcing or deployment process.
 		
-**Post deployment process, look out for database deployment log file which captures all stages of deployment. Log file path is defined in the properties file of the databases.**
-
-**During all the above stages please watch out for any ERRORs which will be capture in the log file.**
-
-Kindly ignore **NOTICE** or **SKIPPING** messages. As these messages states that particular action is already in place hence sql script ignore performing again.
+* Database deployment log file captures all stages of deployment. Check Log files for any ERRORs. Ignore **NOTICE** or **SKIPPING** messages as these  states that particular action is already done.
```

Comment:
Database deployment log file captures all stages of deployment. Check the log files for any errors. Ignore **NOTICE** or **SKIPPING** messages as these messages state that the particular action is already done.